packages/mikmod/mikmod.spec

```diff
@@ -0,0 +1,331 @@
+Name:           mikmod
+Version:        3.2.8
+Release:        7%{?dist}
+Summary:        Console music module player
+
+License:        GPLv2 and LGPLv2+
+URL:            http://mikmod.sourceforge.net/
+Source0:        http://downloads.sourceforge.net/mikmod/%{name}-%{version}.tar.gz
+
+#BuildRequires:  gcc
+#BuildRequires:  ncurses-devel
```

Comment:
Both gcc and ncurses-devel should be fine as build time dependencies - lets leave them active. It's ok if a `.spec` and the `.spec.origfedora` are the same file - it just means no customisation was necessary - but maybe we customise later.